dali/kernels/imgproc/paste/paste_gpu.h

```diff
@@ -36,6 +37,7 @@ namespace paste {
 template<class InputType, int ndims>
 struct PatchDesc {
   const InputType *in;
+  int in_sample_idx;
```

Comment:
Well, this _probably_ could be skipped or there could be a union in/sample_idx, but that would be uglyish.